python/lang/correctness/is-function-without-parentheses.yaml

```
@@ -0,0 +1,17 @@
+rules:
+- id: is-function-without-parentheses
+  languages:
+  - python
+  message: Is "$FUNC" a function or an attribute? If it is a function, you may have
+    meant $X.$FUNC() because $X.$FUNC is always true.
+  patterns:
+  - pattern: $X.$FUNC
+  - pattern-not-inside: $X.$FUNC()
```

You may wish to use ellipses inside the parens here - I can imagine some unconventional `is_*` functions that take arguments

---

I think you are right! 
```suggestion
  - pattern-not-inside: $X.$FUNC(...)
```